projects/angular/src/emphasis/alert/STYLES.md

```diff
@@ -0,0 +1,130 @@
+| --clr-app-alert-danger-action-active-color       | Changes app level danger alert action font color on focus, hover, active.  |
+| --clr-app-alert-danger-close-icon-color          | Changes app level info alert close icon color.                             |
+| --clr-app-alert-danger-close-icon-opacity        | Changes app level info alert close icon opacity.                           |
+| --clr-app-alert-danger-close-icon-hover-color    | Changes app level info alert close icon color on hover.                    |
+| --clr-app-alert-danger-close-icon-hover-opacity  | Changes app level info alert close icon opacity on hover.                  |
+| --clr-app-alert-success-border-color             | Changes app level success alert border color.                              |
+| --clr-app-alert-success-bg-color                 | Changes app level success alert background color.                          |
+| --clr-app-alert-success-icon-color               | Changes app level success alert icon color.                                |
+| --clr-app-alert-success-font-color               | Changes app level success alert font color.                                |
+| --clr-app-alert-success-action-color             | Changes app level success alert action font color.                         |
+| --clr-app-alert-success-action-active-color      | Changes app level success alert action font color on focus, hover, active. |
+| --clr-app-alert-success-close-icon-color         | Changes app level info alert close icon color.                             |
+| --clr-app-alert-success-close-icon-opacity       | Changes app level info alert close icon opacity.                           |
+| --clr-app-alert-success-close-icon-hover-color   | Changes app level info alert close icon color on hover.                    |
+| --clr-app-alert-success-close-icon-hover-opacity | Changes app level info alert close icon opacity on hover.                  |
+| --clr-app-alert-pager-text-color                 | Changes alert pager default font color.                                    |
+| --clr-app-alert-info-pager-bg-color              | Changes info alert pager font color.                                       |
+| --clr-app-alert-warning-pager-bg-color           | Changes warning alert pager font color.                                    |
+| --clr-app-alert-danger-pager-bg-color            | Changes danger alert pager font color.                                     |
+| --clr-app-alert-success-pager-bg-color           | Changes danger success pager font color.                                   |
+
+## Class names
+
+| Class name           | Description                                                              |
+| -------------------- | ------------------------------------------------------------------------ |
+| alert                | Styles an element. All of the following require this class as well.      |
```

Comment:
Remove the boilerplate "Style an element".
"Main alert element" should be enough.